docs/reference-guides/rancher-webhook.md

```diff
@@ -17,11 +17,11 @@ Each Rancher version is designed to be compatible with a single version of the w
 
 <!-- releaseTask -->
 
-| Rancher Version | Webhook Version |
-|-----------------|:---------------:|
-| v2.8.0          |     v0.4.2      |
-| v2.8.1          |     v0.4.2      |
-| v2.8.2          |     v0.4.2      |
+| Rancher Version | Webhook Version | Prime   | Community |
```

Comment:
The first two columns indicate compatibility between versions of different things (Rancher and Webhook). The next two columns indicate if a Rancher version is available through one thing (Prime) or another thing (Community). Suggest renaming column headers to something like "Available in Prime/Community" to make it clear that the last two columns are describing a different relationship than the first two.

Reply:
Updated in https://github.com/rancher/rancher-docs/commit/53357bc7f06fbb67d99f5be7c3a51c6cd866e090.